src/kafka-consumer.cc

```diff
@@ -1406,12 +1407,22 @@ NAN_METHOD(KafkaConsumer::NodeConsume) {
         partition = partitionMaybe.FromJust();
       }
 
+      // Parse onlyApplyTimeoutToFirstMessage
+      Nan::Maybe<bool> onlyApplyTimeoutToFirstMessageMaybe = Nan::To<bool>(info[5].As<v8::Boolean>());  // NOLINT
+
+      bool only_apply_timeout_to_first_message;
+      if (onlyApplyTimeoutToFirstMessageMaybe.IsNothing()) {
+        return Nan::ThrowError("OnlyApplyTimeoutToFirstMessage must be boolean");
+      } else {
+        only_apply_timeout_to_first_message = onlyApplyTimeoutToFirstMessageMaybe.FromJust();
+      }
```

Comment:
I think that `.consume(..)` calls (so not `.consumeFromToppar(..)`) also end up here? Won't `onlyApplyTimeoutToFirstMessageMaybe` be undefined for them? In that case, we should use `false` I think?